README.md

```diff
@@ -153,13 +153,17 @@ superclass. Additionally, it will emit the following custom events.
 
 ### Event: `'flush'`
 
-* No payload
+* [`<Object>`][]
+  * `startPos` [`<Number>`][] - The current file position at the time of data exhaustion
```

Comment:
Should we name this `offset`, `position` or `currentPosition`?

I think it's more clear from the user pov if we remove the `start` prefix.

Reply:
Heh, I wrestled with this and settled on `startPos` for consistency.  I was going to call it `lastReadPosition`, but It felt a little wrong/misdirected to rename something that could be used to instantiate a new instance, e.g. `new TailFile(filname, {startPos: lastReadPosition})`

I agree that it's a little strange as-is, so what's the lesser evil?  Given this, do you still think it should be renamed?  `offset` and `position` feel a little non-descriptive, but I think I like `lastReadPosition` as being the closest name to what it is ("currentPosition" feels too relative).

Reply:
Changed.